rasa/cli/test.py

```diff
@@ -83,6 +84,12 @@ def run_core_test(args: argparse.Namespace) -> None:
     if isinstance(args.model, list) and len(args.model) == 1:
         args.model = args.model[0]
 
+    if args.model is None:
```

Comment:
do we need the same for `train_nlu`?

Reply:
`rasa test nlu` does not fail. In this case `args.model` can not be a list. We use the default model path in case the provided one is not given. So it is already handled.